<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>807</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1035</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Haseln체sse</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="superTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="liveTab">
       <attribute name="title">
        <string>Live</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="CVImageWidget" name="liveView" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="CVImageWidget" name="liveNut" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="rateGroupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>3</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Geschmack</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QRadioButton" name="rateButton1">
                 <property name="text">
                  <string>&amp;1</string>
                 </property>
                 <property name="shortcut">
                  <string>1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rateButton2">
                 <property name="text">
                  <string>&amp;2</string>
                 </property>
                 <property name="shortcut">
                  <string>2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rateButton3">
                 <property name="text">
                  <string>&amp;3</string>
                 </property>
                 <property name="shortcut">
                  <string>3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rateButton4">
                 <property name="text">
                  <string>&amp;4</string>
                 </property>
                 <property name="shortcut">
                  <string>4</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rateButton5">
                 <property name="text">
                  <string>&amp;5</string>
                 </property>
                 <property name="shortcut">
                  <string>5</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Status:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>9</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>... lade ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="NLabel">
            <property name="text">
             <string>N = 0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="text">
             <string>Bild laden</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="captureButton">
            <property name="text">
             <string>Bild aufnehmen</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Zur체cksetzen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="neuralNetCheckBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Benutze Neurales Netz</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="configTab">
       <attribute name="title">
        <string>Konfiguration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTabWidget" name="configTabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="detectTab">
           <attribute name="title">
            <string>N체sse erkennen</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="previousButton">
                <property name="text">
                 <string>Vorheriger Schritt</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+S</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="nextButton">
                <property name="text">
                 <string>N채chster Schritt</string>
                </property>
                <property name="shortcut">
                 <string>Right</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveButton">
                <property name="text">
                 <string>Speichern</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="operationLabel">
              <property name="text">
               <string>Aktuelle Operation:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_1">
                <property name="text">
                 <string>Input:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Output:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="CVImageWidget" name="inputView" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CVImageWidget" name="outputView" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="param1Label">
                <property name="text">
                 <string>Parameter 1:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="param1Slider">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="param1SpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="param2Label">
                <property name="text">
                 <string>Parameter 2:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="param2Slider">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>3</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="param2SpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveConfigButton">
                <property name="text">
                 <string>Konfiguration speichern</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="rateTab">
           <attribute name="title">
            <string>N체sse bewerten</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="previousNut">
                <property name="text">
                 <string>vorherige Nuss</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="nextNut">
                <property name="text">
                 <string>n채chste Nuss</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rate_previousButton">
                <property name="text">
                 <string>Vorheriger Schritt</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+S</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rate_nextButton">
                <property name="text">
                 <string>N채chster Schritt</string>
                </property>
                <property name="shortcut">
                 <string>Right</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rate_saveButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Speichern</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="rate_progressBar">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="rate_operationLabel">
              <property name="text">
               <string>Aktuelle Operation:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Input:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Output:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="CVImageWidget" name="rate_inputView" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CVImageWidget" name="rate_outputView" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="rate_param1Label">
                <property name="text">
                 <string>Parameter 1:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="rate_param1Slider">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="rate_param1SpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="rate_param2Label">
                <property name="text">
                 <string>Parameter 2:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="rate_param2Slider">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>3</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="rate_param2SpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rate_saveConfigButton">
                <property name="text">
                 <string>Konfiguration speichern</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="resultLabel">
              <property name="text">
               <string>Ergebnis:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CVImageWidget</class>
   <extends>QWidget</extends>
   <header>cvimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rate_param1Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rate_param1SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>729</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>733</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rate_param1SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rate_param1Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>728</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rate_param2SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rate_param2Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>845</x>
     <y>726</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>732</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rate_param2Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rate_param2SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>726</y>
    </hint>
    <hint type="destinationlabel">
     <x>840</x>
     <y>719</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
